bot.py

```diff
@@ -1550,22 +1550,42 @@ async def prepembed(ctx, channel:discord.TextChannel, *, jsonInput):
     footerText = jso['footerText'] if 'footerText' in jso else ""
     footerUrl = jso['footerUrl'] if 'footerUrl' in jso else ""
     imageUrl = jso['imageUrl'] if 'imageUrl' in jso else ""
-    embed = assembleEmbed(
-        title=title,
-        desc=desc,
-        titleUrl=titleUrl,
-        hexcolor=hexcolor,
-        webcolor=webcolor,
-        thumbnailUrl=thumbnailUrl,
-        authorName=authorName,
-        authorUrl=authorUrl,
-        authorIcon=authorIcon,
-        fields=fields,
-        footerText=footerText,
-        footerUrl=footerUrl,
-        imageUrl=imageUrl
-    )
-    await channel.send(embed=embed)
+    
+    asWebhook = jso['asWebhook'] if 'asWebhook' in jso else False
+    
+    if asWebhook:
+        embed = assembleEmbed(
+            title=title,
+            desc=desc,
+            titleUrl=titleUrl,
+            hexcolor=hexcolor,
+            webcolor=webcolor,
+            thumbnailUrl=thumbnailUrl,
+            fields=fields,
+            footerText=footerText,
+            footerUrl=footerUrl,
+            imageUrl=imageUrl
+        )
```

Comment:
Why are the `authorName`, `authorUrl`, and `authorIcon` parameters removed from this function call? Those parameters should still be able to be set because they belong to the embed itself.

Reply:
Seems kinda redundant to display the username and pfp on the embed and then again on the webhook. The only other use case is if we want the webhook to display something different to the author of the embed. This probably can be implemented easily ...